apps/admin/src/app/_components/tracks/index.tsx

```diff
@@ -6,14 +6,20 @@ import { ActionBar } from './action-bar';
 import { columns } from './columns';
 import { getTracks } from './tracks.actions';
 
+
 export function ManageTracks() {
   const { data, isLoading } = useQuery(['admin-tracks'], () => getTracks());
-  if (isLoading || !data) return <div>Loading...</div>;
 
   return (
     <div>
-      <ActionBar />
-      <DataTable data={data} columns={columns} />
+      {!data || isLoading ? (
```

Comment:
This effectively doesn't change anything. It just moves the condition from its own if-statement down to the return